plugin/src/main/js/workflow-editor.js

```diff
@@ -54,6 +54,7 @@ $(function() {
                         setTheme(editor);
 
                         if (window.isSystemRespectingTheme) {
+                            // eslint-disable-next-line no-unused-vars
                             window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', event => {
```

Comment:
Or just clean up now. Maybe the unused variable should just be removed?

```suggestion
                            window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', () => {
```

Reply:
Sure, I was just unsure whether `addEventListener` would accept a no-arg function. https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#the_event_listener_callback seems so.